src/index.tsx

```diff
@@ -259,6 +259,11 @@ const LegoEditor = (props: Props): JSX.Element => {
       : initialValue,
   );
 
+  useEffect(() => {
+    const event = new CustomEvent('editorReady');
+    window.dispatchEvent(event);
+  }, []);
+
```

Comment:
Actually on second thought I'm not sure. I think maybe the Slate-component needs some time to initialize, and this wouldn't take that into account

Reply:
Not sure if it'll work 100% either, but we might as well test it out and see how it works :shipit: tried researching for other options but can't seem to find many better alternatives